internal/arrowutil/vector_values.gen.go

```diff
@@ -0,0 +1,601 @@
+// Generated by tmpl
+// https://github.com/benbjohnson/tmpl
+//
+// DO NOT EDIT!
+// Source: vector_values.gen.go.tmpl
+
+package arrowutil
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/array"
+	"github.com/influxdata/flux/memory"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+)
+
+func NewVectorValue(arr array.Interface, typ flux.ColType) values.Vector {
```

Comment:
I think this should be in the values package but I'm assuming there's a reason it's not?